README.md

```diff
@@ -29,30 +28,40 @@ The command line tool has the following options
 Usage: ldnode [options]
 
 Options:
-   -v, --verbose           Print the logs to console
-   --version               Print current ldnode version
-   -m, --mount             Where to mount Linked Data Platform (default: '/')
-   -r, --root              Root location on the filesystem to serve resources
-   -p, --port              Port to use
-   -c, --cache             Set cache time (in seconds), 0 for no cache
-   -K, --key               Path to the ssl key
-   -C, --cert              Path to the ssl cert
-   --webid                 Enable WebID+TLS authentication
-   -s, --secret            HTTP Session secret key (e.g. "your secret phrase")
-   -P, --proxy             Use a proxy on example.tld/proxyPath
-   --no-live               Disable live support through WebSockets
-   -sA, --suffix-acl       Suffix for acl files (default: '.acl')
-   -sM, --suffix-meta      Suffix for metadata files (default: '.meta')
-   -sE, --suffix-sse       Suffix for SSE files (default: '.events')
+   -v, --verbose               Print the logs to console
+   --version                   Print current ldnode version
+   -m, --mount                 Where to mount Linked Data Platform (default: '/')
+   -r, --root                  Root location on the filesystem to serve resources
+   -p, --port                  Port to use
+   -K, --key                   Path to the ssl key
+   -C, --cert                  Path to the ssl cert
+   --webid                     Enable WebID+TLS authentication
+   -s, --secret                HTTP Session secret key (e.g. "your secret phrase")
+   -fU, --force-user           Force a WebID to always be logged in (usefull when offline)
+   -P, --proxy                 Use a proxy on example.tld/proxyPath
+   --no-live                   Disable live support through WebSockets
+   -sA, --suffix-acl           Suffix for acl files (default: '.acl')
+   -sM, --suffix-meta          Suffix for metadata files (default: '.meta')
+   -sE, --suffix-sse           Suffix for SSE files (default: '.events')
+   --no-error-pages            Disable custom error pages (use Node.js default pages instead)
+   --error-pages               Folder from which to look for custom error pages files (files must be named <error-code>.html -- eg. 500.html)
+   --skin                      URI to a skin to load (default: https://linkeddata.github.io/warp/#/list/)
```

Comment:
Skin is a confusing word, but it's important to capture the idea that sometimes an app is invoked as a viewer for particular resource, as opposed to its usual mode.   "Viewer" isn't right, though, because it doesn't have to be read-only.    It's like how operating systems pick some app to run when you indicate you want to open a file (e.g.by double clicking).   What do you call that app?   The "default opener" or something.